### PR TITLE
Create image

### PR DESCRIPTION
**Pull request by ChrisScottThomas**
Creates Packer image using `dwpdigital/awscli` as a base.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ on:
   push:
     branches:
       - master
-    paths-ignore:
-      - 'docs/**'
-      - '*.md'
+    paths-include:
+      - "Dockerfile"
```
Comment (dan-hill2802) [inline .github/workflows/main.yml:8]: This seems harder to maintain if files are added to the repo, people won't think to update this list

File: assume-role [hunk 2]
```diff
@@ -0,0 +1,7 @@
+set +x
```
Comment (dan-hill2802) [inline assume-role]: Do you need this file if you are building from `awscli` image?

Comment (ChrisScottThomas) [reply assume-role]: Nope.  Removed.

File: .github/workflows/main.yml (hunk 1 above)
Comment (ChrisScottThomas) [reply .github/workflows/main.yml:8]: There's no reason to expand this image any further IMO.

Comment (dan-hill2802) [reply .github/workflows/main.yml:8]: there will probably be a `.snyk` file appear at some point

Comment (ChrisScottThomas) [reply .github/workflows/main.yml:8]: But that won't rebuild the image?

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**